text/0000-create-repo-issues.md

```diff
@@ -0,0 +1,103 @@
+- CNB Pull Request: (leave blank)
+- CNB Issue: (leave blank)
+- Supersedes: [#04](https://github.com/buildpacks/rfcs/blob/main/text/0004-rfc-process.md) (which was [rfc#7](https://github.com/buildpacks/rfcs/pull/7))
+
+# Summary
+[summary]: #summary
+Once an RFC has been accepted, prior to it being merged in to the [RFC repo](http://github.com/buildpacks/rfcs), the appropriate sub-team maintainers should create issues in the relevant Buildpacks repositories, linking them to the RFC pull request (using [Github references](https://docs.github.com/en/github/writing-on-github/autolinked-references-and-urls#issues-and-pull-requests))
+
+# Definitions
+[definitions]: #definitions
+
+* RFC &rarr; Request For Comments. It is the process followed by the Cloud Native Buildpacks project, as well as many other OSS ones, to provide a consistent and controlled path for new features to enter the spec, libraries, or tools, so that all stakeholders can be confident about the direction Cloud Native Buildpacks is evolving in.
+* FCP &rarr; Final Comment Period. This is step taken on a particular RFC when enough discussion of the tradeoffs have taken place and the team is in a position to make a decision. Before entering FCP, super majority of the team must sign off.
+* PR &rarr; (Github) Pull Request.
+
+# Motivation
+[motivation]: #motivation
+
+As the number of RFCs increase, it has become increasingly hard to ensure that a given RFC has been implemented, or will be implemented. This RFC proposes an amendment to the [RFC process](https://github.com/buildpacks/rfcs#rfc-process).
+
+The current RFC process [defines](https://github.com/buildpacks/rfcs#rfc-process) that:
+> the team member who merges the pull request should do the following ... Create a corresponding issue in the appropriate repo.
+
+In practice, that adds extra work for the team member who merges in the PR, and it may slip through the cracks occasionally. Additionally, the sub-team maintainers are often the best people to create the issue, given their insight into how it should be implemented.
+
+As an additional consideration, it is often hard to track work from the RFC proposal to the implementation, and difficult to ensure that it was completed. If we use [Github references](https://docs.github.com/en/github/writing-on-github/autolinked-references-and-urls#issues-and-pull-requests) to link the RFC PR to the issue on the appropriate repository, we can better track the work, and be notified when it is fully implemented.
+
+# What it is
+[what-it-is]: #what-it-is
+This defines a process for sub-team maintainers to open issues on the relevant repositories, after FCP for the RFC concludes, before it is merged in.
+
+# How it Works
+[how-it-works]: #how-it-works
+When a PR has been accepted (after FCP has concluded), the sub-team maintainers make issues (tagging the RFC PR) in the relevant buildpack repos, and labels the PR `issues-created`. At that point, the RFC PR may be merged in.
```

Comment:
Given that some sub-teams have multiple maintainers, there may still be some potential for ambiguity around who will handle a particular RFC. I propose that when we move RFCs into FCP we should assign a sub-team maintainer to the PR and they will be responsible for creating the issues.

Reply:
@ekcasey my thought is that this would happen during the weekly subteam meetings, but I don't feel strongly about it.

Reply:
@ekcasey I don't know whether we need that level of process at this point. I agree with @hone that I had felt it could be done during the subteam meetings, as one of the repeating action items. Should I clarify that?

Reply:
There are some RFCs that require multiple sub-teams to create issues from a single RFC. For the sake of minimizing coordination could we maybe change this to be a label per subteam.

For example:

* `issues-created/platform`
* `issues-created/implementation`

If no issues need to be created for a given subteam they can add that label for brevity sake. We can further automate this process by checking that these labels are present.

Reply:
I incorporated @jromero's feedback. LMK what you think.